Cargo.toml

```diff
@@ -32,3 +32,46 @@ resolver = "2"
 # See function names in profiling reports.
 # 2/true is too much, 0 is not enough, 1 is just right for back traces
 debug = 1
+
+[workspace.dependencies]
```

Comment:
Please alphabetize the dependencies. I would probably also just remove the grouping.

Reply:
Done.

Reply:
The `CI / docs` failed because of:
> Error: API rate limit exceeded for 40.76.119.131. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

I don't think I have permission to retry that.